### ISSUE TITLE
Error on during brew update / brew doctor

### ISSUE THREAD

**Issue by malford** [rhyeal/aws-rotate-iam-keys#32]
Tried un-tapping and re-tapping the keg. Seems like branch `develop` was deleted?

```
$ brew tap rhyeal/aws-rotate-iam-keys https://github.com/rhyeal/aws-rotate-iam-keys
Updating Homebrew...
==> Tapping rhyeal/aws-rotate-iam-keys
Cloning into '/usr/local/Homebrew/Library/Taps/rhyeal/homebrew-aws-rotate-iam-keys'...
remote: Enumerating objects: 44, done.
remote: Counting objects: 100% (44/44), done.
remote: Compressing objects: 100% (38/38), done.
remote: Total 44 (delta 3), reused 28 (delta 1), pack-reused 0
Unpacking objects: 100% (44/44), 82.68 KiB | 1.50 MiB/s, done.
fatal: Not a valid object name develop
Tapped 1 formula (101 files, 286.6KB).
```
```
$ brew doctor
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

fatal: Not a valid object name develop
```

**Comment by jasoniangreen** [comment]
IS this a duplicate of #29 ?

**Comment by malford** [comment]
I don't think so. I get the same error when running the cmd `git describe develop` in the repo which is used here: https://github.com/rhyeal/aws-rotate-iam-keys/blob/11761486c3ffceb76741c134e05814fc9db5f34d/Formula/aws-rotate-iam-keys.rb#L19

**Comment by marianol** [comment]
Running into this same issue "tap" as well as #29 on "install"

**Comment by mmrwoods** [comment]
@malford @marianol @jasoniangreen Sorry, ~I think~ this was my fault... we use a forked version of `aws-rotate-iam-keys` and submit PRs once we're confident our `develop` version is working as expected. 

I added the head and develop software specs to support this workflow, submitted a PR wihout testing what happens when there is no `develop` branch, and @rhyeal merged my PR.

I'll fix this and submit a PR for @rhyeal. I'm pretty sure this is just a warning though, it should not prevent you installing the package (as long as you're not trying to install `--devel`). 

There is a separate issue with the checksum being incorrect, which will prevent you installing the default/stable version. You can work around this until a new release is published with a valid checksum by installing from the latest HEAD version instead of the stable release, e.g. 

`brew install aws-rotate-iam-keys --HEAD`

**Comment by rhyeal** [comment]
Agreed that this is not related to #29 

I'll review the PR and merge in as I can. Horray more time to work from home the next few weeks!

**Comment by malford** [comment]
New Homebrew error:
```
Warning: Calling 'devel' blocks in formulae is deprecated! Use 'head' blocks or @-versioned formulae instead.
Please report this issue to the rhyeal/aws-rotate-iam-keys tap (not Homebrew/brew or Homebrew/core), or even better, submit a PR to fix it:
  /usr/local/Homebrew/Library/Taps/rhyeal/homebrew-aws-rotate-iam-keys/Formula/aws-rotate-iam-keys.rb:21

fatal: Not a valid object name develop
```

**Comment by mmrwoods** [comment]
@marianol @malford @jasoniangreen Finally fixed in version 0.9.8.1, please run `brew update`.
